xml/security_preface.xml

```diff
@@ -8,753 +8,450 @@
-     program to live. Instead, they are specialized to spread as quickly as
-     possible on network structures. The worms that appeared in the past,
-     such as Ramen, Lion, or Adore, used well-known security holes in
-     server programs like bind8. Protection against worms is relatively
-     easy. Given that some time elapses between the discovery of a security
-     hole and the moment the worm hits your server, there is a good chance
-     that an updated version of the affected program is available on time.
-     That is only useful if the administrator actually installs the security
-     updates on the systems in question.
+     Use strong authentication for network services. For details, see
+     <xref linkend="part.auth" />.
     </para>
-   </sect3>
-  </sect2>
+   </listitem>
+  </itemizedlist>
  </sect1>
- <sect1 xml:id="sec.security.general.tips">
-  <title>Some General Security Tips and Tricks</title>
-
-  <remark condition="clarity"> 2007-07-27 - jjaeger: please use profiling for mailing lists and
-   websites if they change from SLE to opensuse, next time. </remark>
 
+ <sect1 xml:id="sec.security.vulnerabilities">
+  <title>Software Vulnerabilities</title>
   <para>
-   To handle security competently, it is important to observe some
-   recommendations. You may find the following list of rules useful in
-   dealing with basic security concerns:
+   Software vulnerabilities are a class of software bugs that can be
+   exploited by attackers to obtain unauthorized access to data or use
+   attacked systems in unintended ways. Especially for remote services,
+   for example HTTP servers, vulnerabilities can be very critical. In
+   current computer environments, vulnerabilities have to be expected.
+   Usually it is the obligation of programmers to fix these issues when
+   they become known. It is then the administrators task to quickly and
+   safely install provided updates to protect the systems.
   </para>
-
-  <remark>2011-03-17 jw: Most users are surprised to find plain text copies of a password in their home.
-  Should we explain when that happens, and how to avoid it? E.g. .netrc, .subversion/auth/*, .oscrc, et al.
-  </remark>
-
-  <itemizedlist mark="bullet" spacing="normal">
+  <para>
+   Vulnerabilities are usually announced on centralized
+   databases, for example the <emphasis>National Vulnerability
+   Database</emphasis>, which is maintained by the US government.
+   You can subscribe to feeds to stay informed about newly
+   discovered vulnerabilities. In some cases the problems induced by
+   the bugs can be mitigated until a software update is provided.
+   Vulnerabilities are assigned a <emphasis>Common Vulnerabilities
+   and Exposures</emphasis> (<emphasis>CVE</emphasis>) number and a
+   <emphasis>Common Vulnerability Scoring System</emphasis>
+   (<emphasis>CVSS</emphasis>) score. The score helps identifying
+   the severity of vulnerabilities.
```

Comment:
Suggestion: `<para>Vulnerabilities are usually announced through centralized databases. The most well-known is the <emphasis>National Vulnerability Database</emphasis> which is maintained by the US government. Vulnerabilities are assigned a <emphasis>Common Vulnerabilities and Exposures</emphasis> (<emphasis>CVE</emphasis>) number and a <emphasis>Common Vulnerability Scoring System</emphasis> (<emphasis>CVSS</emphasis>) score. In some cases the problems induced by the bugs can be mitigated until a software update is provided.<para><para>To stay informed about newly discovered vulnerabilities, subscribe to the CVE feeds provided at <link xlink:href="https://cve.mitre.org/cve/data_updates.html"/>.`